microsoft-365/managed-desktop/get-started/register-devices-partner.md

```diff
@@ -17,69 +17,41 @@ This topic describes the steps for Partners to follow to register devices. The p
 
->[!NOTE]
->This format is only for the Partner process. The process for self-registration is documented in [Register devices in Microsoft Managed Desktop yourself](register-devices-self.md).
 
 >[!IMPORTANT]
->These values must match the manufacturer values from SMBIOS exactly, including capitalization and special characters. 
+>The Group Name must match exactly **Microsoft365Managed_Autopilot** including capitalization and special characters. This will allow the newly registered devices to be assigned with the Microsoft Managed Desktop Autopilot profile.
 
-- Device manufacturer (example: SpiralOrbit) 
-- Device model (example: ContosoABC)
-- Device serial number
+>[!NOTE]
+> You should have received this .csv file with your device purchase. If you didn't receive a .csv file, you can create one yourself by following the steps in [Adding devices to Windows Autopilot] (https://docs.microsoft.com/windows/deployment/windows-autopilot/add-devices#collecting-the-hardware-id-from-existing-devices-using-powershell). The Windows PowerShell Ssript is different from the one used for the [Microsoft Managed Desktop Admin portal] (https://docs.microsoft.com/microsoft-365/managed-desktop/get-started/register-devices-self?view=o365-worldwide#obtain-the-hardware-hash). Partners should use [Get-WindowsAutoPilotInfo] (https://www.powershellgallery.com/packages/Get-WindowsAutoPilotInfo) to register devices for Microsoft Managed Desktop devices in Partner Center.
 
-## Register devices by using the Azure Portal
+If you get an error message while trying to upload the .csv file, check the format of the file. You can use the hardware hash only, or the OEM name, serial number, and model (in that column order), or the Windows Product ID. You can also use the sample .csv file provided from the link next to **Add devices** to create a device list. 
 
-Registering by using the Azure Portal is the same as for self-service, except you access the portal differently. Follow these steps:
+For more information about Autopilot in Partner scenarios, see [Add devices to a customer’s account] (https://docs.microsoft.com/partner-center/autopilot#add-devices-to-a-customers-account).
+
+
+## Register devices by using the OEM API
+
+Before completing registration for a customer, you must first establish a relationship with them. You should have a unique link to provide to your respective customers. See [How to establish OEM relationship] (https://docs.microsoft.com/windows/deployment/windows-autopilot/registration-auth#oem-authorization).
+
+Once you've established the relationship, you can start registering devices for customers using the Group Tag **Microsoft365Managed_Autopilot**.
+
+>[!IMPORTANT]
+>The group name must match exactly **Microsoft365Managed_Autopilot** including capitalization and special characters. This will allow the newly registered devices to be assigned with the Microsoft Managed Desktop Autopilot profile.
```

Comment:
```suggestion
> The group name must **Microsoft365Managed_Autopilot** exactly, including capitalization and special characters. This will allow the newly registered devices to be assigned with the Microsoft Managed Desktop Autopilot profile.
```